frontend/javascripts/admin/task/task_create_form_view.js

```diff
@@ -68,46 +70,88 @@ type State = {
 };
 
 export function handleTaskCreationResponse(responses: Array<TaskCreationResponse>) {
+  if (responses.length === 0) {
```

Comment:
When does this happen? Sounds like a case which should never happen? (in that case, I'd rather throw an error here)

Reply:
You are correct. This should never happen :slightly_smiling_face: